### DISCUSSION TITLE
Little help with versioning 🙏

### DISCUSSION THREAD

**Discussion by fgatti675** [facebook/docusaurus#7510] [Q&A]
Hi! Not sure if I've hit a bug but I can't get versioning to work as I intend to.
So I have my v1.0.0 docs under the folder `versioned_docs/version-1.0.0` and my v2.0.0 under simply `docs`
I would like to make it so that : 
- when user types `/docs` uses 1.0.0 by default
- when user types `/docs/2.0.0` uses 2.0.0 by default
My `versions.json `looks like this:
```
[
  "1.0.0",
  "current"
]
```
And my docs section of `docusaurus.config.js`:
```
    docs: {
        sidebarPath: require.resolve('./sidebars.js'),
        lastVersion: "current",
        editUrl: "https://github.com/Camberi/firecms/tree/website/website",
        versions: {
            "1.0.0": {
                label: '1.0.0',
                path: '/'
            },
            current: {
                label: '2.0.0-alpha',
                path: '2.0.0',
                banner: "unreleased"
            }
        },
    },
```           
With this config, the routes under `/docs/` work as expected but under `/docs/2.0.0/` is 404.
If I change the path of 1.0.0 to `1.0.0` instead of `/` everything works as expected, which has me pretty confused.  
What am I doing wrong?? Thanks!!

**Comment by slorber** [comment]
It's difficult to help you without a repro: please take the time to create one showing your problem in action

**Comment by fgatti675** [comment]
Thanks, I got it to work. I found it pretty confusing though.
This is my config in case it helps:
```
                docs: {
                    sidebarPath: require.resolve('./sidebars.js'),
                    lastVersion: "1.0.0",
                    editUrl: "https://github.com/Camberi/firecms/tree/website/website",
                    versions: {
                        "1.0.0": {
                            label: '1.0.0',
                        },
                        current: {
                            label: '2.0.0-alpha',
                            path: '2.0.0',
                            banner: "unreleased"
                        }
                    },
                },
```

**Reply by slorber** [reply]
What is confusing, what was the problem and how can we improve?
